### DISCUSSION TITLE
[EN DatetimeV2] Add support for exclusions like "not on Saturday"

### DISCUSSION THREAD

**Discussion by bfcamara** [microsoft/Recognizers-Text#1919] [Ideas]
**Is your feature request related to a problem? Please describe.**
I'm building a bot that works heavily with dates and times, and we need to have a way to identify exclusions.

For example: "anytime but Wednesdays", "not on Friday", "except weekend"

**Describe the solution you'd like**
It would be great if the date recognizer could return in its ModelResult the indication that it's an exclusions

**Describe alternatives you've considered**
Apply a regex after having the model result to detect a set of common expressions for exclusions (not on, anything but, etc.)

**Additional context**
N/A

**Comment by tellarin** [comment]
@bfcamara, we see the usefulness of that and it would indeed be interesting to have support for it (in the recognizers or another module). But unfortunately this is not currently in the plans for the recognizers in the mid-term (as it can get quite complex outside basic cases).

**Comment by bfcamara** [comment]
@tellarin Thank you for your reply. I totally agree with about the complexity of this feature. That might explain why (afaik) well known libs (SUTime, GuTime, HeidelTime )don't support it either.
